src/conftest.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+# -*- coding: utf -8 -*-
+"""File with the fixture configuration"""
+
+
+import pytest
+
+
+from queue import Queue
+
+
+@pytest.fixture(scope='function')
+def q0():
+    return Queue()
```

Comment:
Glad to see you guys are starting to use fixtures! However, you will need to expand the diversity of cases. What about a Queue with only one item in it? What about two items? What about a thousand items in it?